### DISCUSSION TITLE
Unable to stop Carousel on large screen

### DISCUSSION THREAD

**Discussion by TheBuzzer67** [twbs/bootstrap#36756] [General] [answered]
I used to show 3 image on my main page of web site and to stop the carousel, I used to put active on all item but since BS5, it still turning the images and looks like s..t on screen.

I tried [https://stackoverflow.com/questions/71154792/how-to-stop-bootstrap-5-carousel-autoslide-only-for-large-screens/73008238#73008238](these suggestions) and still not work cause if I put interval to false, it is spinning faster than ever.

I tried the .carousel('pause') not work.

What is the suggestion?

Do I have to start an issue?

**Comment by TravelTimN** [comment, marked as answer]
@TheBuzzer67 would you be able to provide a repository or pen somewhere with your code, so we can see what you've attempted so far? That way we can try to help identify your issue to get your carousel working as expected.

**Comment by TheBuzzer67** [comment]
Hi!

Pausing wasn't enough and let the carousel restart on hover or after a certain delay! 

So I found that I got to unset all values before the pausing/cycling.

[https://stackoverflow.com/questions/71154792/how-to-stop-bootstrap-5-carousel-autoslide-only-for-large-screens/](Same issue solved with my code)

So for me, it is resolved.

**Reply by TravelTimN** [reply]
Ah excellent! Glad you got it working, especially since the original poster from that link said they couldn't get it working. Well done! 👍🏻

**Reply by AlexNzMe** [reply]
@TheBuzzer67 would you mind please to post your solution here, I'm facing similar issue and unable to stop the carousel with JavaScript.
